python/lsst/ts/observatory/control/utils/remote_group_test_case.py

```diff
@@ -146,6 +146,8 @@ async def wait_for(self, coro, timeout, description, verbose):
             print(f"wait for {description}")
         try:
             result = await asyncio.wait_for(coro, timeout=timeout)
+        except asyncio.CancelledError:
+            print(f"{description}: Cancelled error.")
```

Comment:
I suggest re-raising the cancelled error, as that more closely mimics `asyncio.wait_for`. Or if that causes problems then please explain in the doc string what you do with it.

Reply:
I think this got here by mistake... I was debugging a `CancelledError` exception that I get from time to time and I can't figure out where it is coming from.

Reply:
Ah, OK. I always try to remember to catch CancelledError if I'm catching other errors as well, since I usually want to treat cancellation specially (e.g. quit a loop, which might continue for other errors). So I assumed you were doing that here.